src/auth/auth-api-request.ts

```diff
@@ -2035,7 +2058,7 @@ const CREATE_TENANT = new ApiSettings('/tenants', 'POST')
  */
 export class AuthRequestHandler extends AbstractAuthRequestHandler {
 
-  protected readonly tenantMgmtResourceBuilder: AuthResourceUrlBuilder;
+  protected readonly v2ResourceBuilder: AuthResourceUrlBuilder;
```

Comment:
What's the benefit changing the name here?

Reply:
seems like authRequestHandler works with all V2 apis, it will be duplicate to implemented another authHandler.  With the naming `tenantMgmtResourceBuilder` it seems narrow down it's usage. Hence, renaming to be used by getProjectConfig/updateProjectConfig APIs.  For a specific tenant level access, there is another authHandler called `TenantAwareAuthRequestHandler` which left un-touched.

Reply:
Let's call it `authResourceUrlBuilder` then, so that we don't need to update the name when we starts using v3

Reply:
Please update the comments of the class as well